src/lib/components/Sidebar/SidebarCollapse.tsx

```diff
@@ -11,12 +11,18 @@ import { SidebarItemContext } from './SidebarItemContext';
 
 export type SidebarCollapseProps = PropsWithChildren<ComponentProps<'button'> & SidebarItemProps>;
 
-const SidebarCollapse: FC<SidebarCollapseProps> = ({ children, icon: Icon, label, ...props }): JSX.Element => {
+const SidebarCollapse: FC<SidebarCollapseProps> = ({
+  children,
+  icon: Icon,
+  label,
+  defaultIsOpen = false,
+  ...props
+}): JSX.Element => {
   const theirProps = excludeClassName(props);
 
   const id = useId();
   const { isCollapsed } = useSidebarContext();
-  const [isOpen, setOpen] = useState(false);
+  const [isOpen, setOpen] = useState(defaultIsOpen);
```

Comment:
Wouldn't be more interesting to enable more than a default open status but a possibility to control externally the status?

having : 
open = false as prop

```
const [isOpen, setOpen] = useState(open);
```

```
useEffect(() => setOpen(open), [open]
```

if none prop is passed, no effect is possible and the internal state runs its way. But having an external control would be welcome for cases we would like to use an external CTA to open/close the sidebar.

What do you think?

Reply:
I can do that. I'm in favor of controlled components. I've been unavailable for the last couple weeks and will be for the next week, but I'll make the change and play around with it. I still need to write a test for this, as well. I'm unfamiliar with the testing libraries in this repo, so I'll need to take some time looking at other component tests.

Reply:
@rluders @CYB3RL1F3 I apologize, I'm going to abandon this PR since I've had to keep putting it off due to more urgent work, and ultimately, due to flowbite/flowbite-react not currently working with Turborepo, I've been forced back to bootstrap for the time being, so I don't think I would get back to this any time soon.

Reply:
What is missing here to get it done? Maybe I can assume it.

Reply:
If kept as is, then just the tests, unless you want to implement @CYB3RL1F3's suggestions

Reply:
OK, thanks. I can continue this one tomorrow. :)

Reply:
Thank you, sir. I apologize again. If in the future I find flowbite workin with Turbo, I'll be back. Really interested in the platform, just on a time crunch, unfortunately